index.html

```diff
@@ -0,0 +1,147 @@
+                    <li><a href="">English</a></li>
+                </ul>
+            </div>
+        </div>
+        <!-- Desktop Navbar -->
+        <div class="desktop-navbar">
+            <div class="logo">
+                <img src="./images/logo.png" alt="logo-image">
+            </div>
+            <div class="desktop-nav-list">
+                <ul>
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="#services-id">Our Services</a></li>
+                    <li><a href="#team-id">Team Members</a></li>
+                    <li><a href="#team-id">Sponser</a></li>
+                </ul>
+            </div>
+        </div>
+        </nav>
+        <div class="description">
+            <h1>Building Sustainable Future Infrastructures</h1>
+            <p id="detail">We are well established company and has a very strong technical profile of engineering,
+                procurement, and construction of power grid stations and transmission lines.</p>
+            <h3>2022.10.17(Mon) ~ 21 (Fri)</h3>
+            <p>@ Expo Center Lahore, Pakistan</p>
+        </div>
+    </section>
+
+    <!-- Homepage section Ends -->
+
+    <!-- Program Section start -->
+    <section id="services-id" class="program">
+        <h2 id="services-title">Our Services</h2>
+        <div id="redline"></div>
+        <div class="service-cards">
+            <div class="card">
+                <img src="./images/civil infrastructure.png" class="card-images" alt="">
+                <h3 id="program-title">Civil Infrastructure</h3>
+                <p>The development of civil infrastructure is always our key priority.</p>
+            </div>
+            <div class="card">
+                <img src="./images/renewable-energy-icon.webp" class="card-images" alt="">
+                <h3 id="program-title">Renewable Energy</h3>
+                <p>Building sustainable infrastructures is part of our vision.</p>
+            </div>
+            <div class="card">
+                <img src="./images/trading-icon.png" class="card-images" alt="">
+                <h3 id="program-title">Trading Activities</h3>
+                <p>Along with EPC solutions, we are also providing consultancy services to international firms.</p>
+            </div>
+            <div class="card">
+                <img src="./images/erection of equipment.png" class="card-images" alt="">
+                <h3 id="program-title">Erection of Equipment </h3>
+                <p>The process of assembling all components of equipments.</p>
+            </div>
+            <div class="card">
+                <img src="./images/commisoing icon.png" class="card-images" alt="">
+                <h3 id="program-title">Testing and Commissioning</h3>
+                <p>Testing and commissioning of installed equipment so that they are safe and meet design requirement.
+                </p>
+            </div>
```

Comment:
Kindly ensure that all images in this and other parts of  your project do have appropriate values for the alt attribute. This is very important since it has many benefits including ease of use for screen readers.

Reply:
Alt attribute added